README.md

```diff
@@ -5,6 +5,15 @@ GitHub page: https://amsterdam.github.io/afkortingen/
 Abbreviations used in the City of Amsterdam and the DataPunt organization.
 Here is a none exaustive list of abbrevations alongside possible translations and external/internal resources for new developpers to get aquainted quickly to our ever expanding list of abbreviations.
 
+# New abbreviation?
+
+If you come accross an abbreviation, and it is already widely used in the gemeente or in your domain, then add it.
+
+If you have come up with a great new abbrevation yourself, then please, don't just add it.
+Discuss it with colleges whether the benefit of a few less syllables/letters outway the cost of requiring this additional knowledge in order to function in the organization.
+
+See the ASS rule: https://gist.github.com/klaaspieter/12cd68f54bb71a3940eae5cdd4ea1764
```

Comment:
Maybe add the meaning of the acronym; "Acronyms Seriously Suck"

Reply:
Agreed, updated